### ISSUE TITLE
generates SRT from video 

### ISSUE THREAD

**Issue by lenmao** [Const-me/Whisper#124]
I'm really perplexed and don't know whether to laugh or cry. Earlier, I used Whisper to convert videos and generate Chinese SRT subtitles from Englisth input, and English SRT subtitles from English input. 

However, for the past two days, no matter what I specify, it only generates English SRT subtitles. The audio in the videos is in English. Has anyone else encountered a similar issue?

**Comment by lenmao** [comment]
I can't quite figure it out now. I have an MP4 video that is over 9 hours long. I'm certain that I used either Buzz or Whisper desktop to convert the English in this video into two separate SRT files: one in Chinese and one in English. However, no matter how I try now, I can't replicate the process, and it seems that Whisper doesn't support this feature. I'm wondering how I managed to do it back then.

**Comment by emcodem** [comment]
Yes, whisper has not been trained to translate from english to chinese but it can by accident output chinese subtitles for english audio in case it has accidently been trained "with wrong  data". IF whisper translates english to chinese, it is accident or better wrong trainging; a consequence of the weak supervised training according to the original documentation
